### PR TITLE
VMware - virtual machines files.

### LINKED ISSUE #311: Add VMware content [4]
basic VMware ops

### PR DESCRIPTION
**Pull request by mihaitusa**
Closes #311 
Signed-off-by: Mihail-Cristian Tusa <mihail-cristian.tusa@hpe.com>

### CODE REVIEW

**Comment by gituser173** [comment]
@mihaitusa looks good :+1:

**Comment by mbettan** [comment]
Any updates when it will be accepted ? @meshipeer  @mihaitusa 

I would like to update my customer very interested by this content. Thanks

**Comment by mbettan** [comment]
I think it should be inside "virtualization" folder, it's not a Cloud Provider like AWS

**Comment by mbettan** [comment]
Any plan for VMware Clone Virtual Machine ? it's the most important operation, ability to clone a template VMware?

**Comment by mihaitusa** [comment]
@mbettan  We should discuss offline the the Clone Virtual Machine operation topic in order to emphasize the use cases.

**Comment by mihaitusa** [comment]
@Bonczidai Please review it!

**Comment by mihaitusa** [comment]
@mbettan  Take a look upon the /sample/virtual_machine_lifecycle.sl and tell us WDYT.

**Comment by mbettan** [comment]
That's good for my POC scenario - It can show a end to end scenario with VMware

File: content/io/cloudslang/cloud_provider/vmware/samples/virtual_machine_lifecycle.sl [hunk 1]
```diff
@@ -0,0 +1,552 @@
...
+            - trust_everyone
+            - virtual_machine_name
+            - operation
+            - device
+            - vm_disk_size
+            - vm_disk_mode
+        publish:
+          - return_result
+          - return_code
+          - exception : ${exception if exception != None else ''}
```
Comment (Bonczidai) [inline content/io/cloudslang/cloud_provider/vmware/samples/virtual_machine_lifecycle.sl]: `get`?

**Comment by mihaitusa** [comment]
@Bonczidai Please review it!

**Comment by Bonczidai** [comment]
once build passes. we are good to go

**Comment by mbettan** [comment]
Can you merge it please? Would like to communicate on it

**Comment by mbettan** [comment]
Any update?

**Comment by mihaitusa** [comment]
@mbettan  Just hang on a little bit. We are going to move the vmware folder from "cloud_provider" to a new folder "virtualization". Check again at the end of the day... should be ready!

**Pull request merged**